testsuite/LYS_ASP_peptide.py

```diff
@@ -0,0 +1,199 @@
+# Load espresso, pyMBE and other necessary libraries
+import os
+import sys
+import inspect
+import numpy as np
+import pandas as pd
+from tqdm import tqdm
+import espressomd
+from espressomd import interactions
+from espressomd.io.writer import vtf
+from espressomd import electrostatics 
+
+# Create an instance of pyMBE library
+import pyMBE
+pmb = pyMBE.pymbe_library()
+
+# Load some functions from the handy_scripts library for convinience
+from handy_scripts.handy_functions import setup_electrostatic_interactions
+from handy_scripts.handy_functions import minimize_espresso_system_energy
+from handy_scripts.handy_functions import setup_langevin_dynamics
+from handy_scripts.handy_functions import block_analyze
+
+# The trajectories of the simulations will be stored using espresso built-up functions in separed files in the folder 'frames'
+if not os.path.exists('./frames'):
+    os.makedirs('./frames')
+
+# Simulation parameters
+pmb.set_reduced_units(unit_length=0.4*pmb.units.nm)
+pH_range = np.linspace(2, 12, num=21)
+Samples_per_pH = 36
+MD_steps_per_sample = 50
+steps_eq =int(Samples_per_pH/3)
+N_samples_print = 10 # Write the trajectory every 100 samples
+probability_reaction = 0.5 
+SEED = 1
+dt = 0.01
+solvent_permitivity = 78.3 
+
+L = 25.513*pmb.units.nm
+
+# Peptide parameters
+N_aminoacids = 5
+sequence = 'K'*N_aminoacids+'D'*N_aminoacids
+model = '2beadAA'  # Model with 2 beads per each aminoacid
+pep_concentration = 1e-4 *pmb.units.mol/pmb.units.L
+
+# Solution parameters
+cation_name = 'Na'
+anion_name = 'Cl'
+c_salt = 1e-2 * pmb.units.mol/ pmb.units.L
+
+# Define salt parameters
+
+pmb.define_particle( name=cation_name,  q=1, diameter=0.35*pmb.units.nm, epsilon=1*pmb.units('reduced_energy'))
+pmb.define_particle( name=anion_name,  q=-1, diameter=0.35*pmb.units.nm,  epsilon=1*pmb.units('reduced_energy'))
+
+# Load peptide parametrization from Lunkad, R. et al.  Molecular Systems Design & Engineering (2021), 6(2), 122-131.
+
+pmb.load_interaction_parameters (filename=pmb.get_resource('reference_parameters/interaction_parameters/Lunkad2021.txt'))
```

Comment:
We could have `load_interaction_parameters()` and friends internally call `pmb.get_resource()`. But then users wouldn't be able to read reference parameters that lie outside of the pyMBE root folder. This could be an issue if you plan to package pyMBE on PyPi, and could also be an issue in a teaching environment like the summer school, where Python packages are pinned before the school and only accessible in read-only mode.

I have no preference on how this is achieved. I opted for this temporary solution to remove the fragile code that existed before, and didn't manage to write a more elegant solution based on `setuptools.pkg_resources` or `importlib.resources` due to the lack of `__init__.py` file in the pyMBE tree structure.